app/pages/UcsfHomePage/UcsfHomePage.module.scss

```diff
@@ -0,0 +1,62 @@
+@import '~styles/utils/_helpers.scss';
+@import '~styles/components/_ucsf-layout.scss';
+
+.resourceList {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 30px;
+  @media screen and (max-width: $break-tablet-s) {
+    flex-direction: column;
+    align-items: center;
+  }
+}
+
+.resourceItem {
+  width: 160px;
+  height: 160px;
+  margin: 0 15px 15px 0;
+  border-radius: 5px;
+  box-shadow: 0 0 4px rgb(0 0 0 / 20%);
+  &.isChecked {
+    box-shadow: none;
+    border: solid 1px $color-brand;
+  }
+  @media screen and (max-width: $break-tablet-s) {
+    margin-right: 0;
+  }
+}
+
+.resourceLabel {
```

Comment:
The Figma design shows that the text in the label should have a font size of 18px, but because you don't set it here, it ends up being 16px, which is a tiny bit smaller that it's supposed to be.

I'm also seeing that the Figma design chooses a font weight of 500, but we don't have that font size available; the three font weights that we currently download are 400, 600, and 700. We may want to clarify with the design team exactly which font faces we're supposed to be using.